data/a11ySuite.js

```diff
@@ -50,7 +51,13 @@
           a11ySuite.eachTest = function() {
             // instantiate fixture
             fixtureElement.create();
-
+            
+            // Make sure lazy-loaded dom is ready (eg <template is='dom-repeat'>)
+            Polymer.dom.flush();
+            
+            // If we have a beforeEach function, call it
+            if(beforeEach) beforeEach();
```

Comment:
Add braces to one line ifs please. It's more robust to merges, typos, ets. One `goto fail;` per civilization is enough I think :)